app/components/works/edit/attached_file_component.rb

```diff
@@ -18,6 +18,10 @@ def initialize(attached_file:, depth:, form:)
       def uploaded?
         !attached_file.blob.nil?
       end
+
+      def path
+        attached_file.paths.join('/')
+      end
     end
```

Comment:
this name make sense here (given it is joining the paths), though we may want to consider dropping a comment indicating this the path to the file (excluding the filename)....confusion may arise since unfortunately there is a `path` method on `attached_file` which returns a string with the full path including filename

the response could be to change the somewhat poorly named `path` method in `attached_file` too, but that will touch a bunch of other code, so may be besst done in a future refactor

Reply:
Totally agree with adding a comment. `attached file` `path` is actually a db field name so rename is non-trivial.